examples/wget/Kconfig

```diff
@@ -12,11 +12,18 @@ config EXAMPLES_WGET
 
 if EXAMPLES_WGET
 
+config EXAMPLES_WGET_STACKSIZE
+	int "wget stack size"
+	default 2048
```

Comment:
should we use DEFAULT_TASK_STACKSIZE here?

Reply:
Actually, this patch is for SSL support, but useful in the future.

Reply:
Yes, but it's better to change 2048 to DEFAULT_TASK_STACKSIZE, so we can chagne all program stack size(e.g. sim).

Reply:
I misunderstood. I'll fix it.